plugins/dell-kestrel/fu-dell-kestrel-plugin.c

```diff
@@ -256,6 +256,7 @@ fu_dell_kestrel_plugin_config_mst_dev(FuPlugin *plugin)
 	/* flags */
 	fu_device_add_private_flag(device_mst, FU_DEVICE_PRIVATE_FLAG_EXPLICIT_ORDER);
 	fu_device_add_private_flag(device_mst, FU_DEVICE_PRIVATE_FLAG_SKIPS_RESTART);
+	fu_device_add_private_flag(device_mst, FU_DEVICE_PRIVATE_FLAG_ONLY_WAIT_FOR_REPLUG);
```

Comment:
this is already set in the fu_intel_usb4_device_init -- no?

Reply:
Nah, I want to get rid of FU_DEVICE_PRIVATE_FLAG_ONLY_WAIT_FOR_REPLUG in the next couple of releases anyway.